addon/components/base/bs-carousel.js

```diff
@@ -58,13 +138,28 @@ export default Ember.Component.extend({
    * @type string
    */
   directionalClassName: null,
+
   /**
-   * Contains all information to render indicators.
-   *
    * @property indicators
    * @private
    */
-  indicators: null,
+  indicators: computed('_index', 'childSlides', 'showIndicators', function() {
+    if (this.get('showIndicators') === false) {
+      return;
+    }
+    let indicators = [];
+    for (let a = 0; a < this.get('childSlides').length; a++) {
+      indicators.push({
+        active: false,
+        index: a
+      });
+    }
+    if (indicators.length > 0) {
+      indicators[this.get('_index')].active = true;
+    }
+    return indicators;
+  }),
```

Comment:
I think this could be refactored a bit, to make it simpler. And because it depends on `_index`, it will create a new array every time the index changes, so causing a full rerender of all indicators. How about this:

```
indicators: computed('childSlides.length', function() {
  return new Array(this.get('childSlides.length'));
}
```

This would need some changes in the template as well:

```hbs
{{#if showIndicators}}
  <ol class='carousel-indicators'>
    {{#each indicators as |indicator index|}}
      <li class='{{if (bs-eq _index index) 'active'}}' onclick={{action 'toSlide' index}} role='button'></li>
    {{/each}}
  </ol>
{{/if}}
```

This makes use of the existing `bs-eq` helper to set the active class directly in the template, and `#each` has a built-in index, so no need to put that into the array...